eq-author/src/components/ToastContainer/index.js

```diff
@@ -55,36 +38,15 @@ const ToastWrapper = styled.div`
   justify-content: center;
 `;
 
-export const ToastArea = ({
-  toasts,
-  dismissToast,
-  undoToast,
-  ...otherProps
-}) => {
```

Comment:
Don't think you've updated the tests for this component probably a couple broken snapshot tests in there somewhere.

Reply:
Fixed